money.go

```diff
@@ -85,6 +88,33 @@ func New(amount int64, code string) *Money {
 	}
 }
 
+// NewFromFloat creates and returns new instance of Money from a float64.
+func NewFromFloat(amount float64, currency string) *Money {
+	currencyDecimals := math.Pow(10, float64(GetCurrency(currency).Fraction))
+	return New(int64(amount*currencyDecimals), currency)
+}
+
+// NewFromString creates and returns new instance of Money from a string.
+func NewFromString(amount string, currency string) (*Money, error) {
```

Comment:
@npinochet Thanks for the PR!

`NewFromString` is a bit tricky to implement. I can think of so many cases we need to take into account, that I'm not really sure that it is a job for the Money package to crack it. When I see the function called NewFromString I expect that it takes the string my project represents/displays the money and it creates the Money instance from it, but from this implementation it looks like it only accepts a single format (xx.xxx). Moreover, it cannot handle the `Money.Display()` output, which I would expect it to do. Couple of things to consider before implementing it:

Should we clearly document that it only accepts the "number" in a specific format, e.g. "xx.xxx" and nothing else? 
When we display money we sometimes separate thousands with comma, should we handle it too? 
What about the cases where the money has a currency sign in it '£ xx.xxx' or the currency code 'xx.xxx USD'?
Should we trim or remove any other non numeric characters from a string? Or return an error? e.g. "  12.123 ", "12.beefwellington"?

In my opinion, I would leave up to the caller to parse the string to int/float depending on the project needs and then pass it to money to create a new instance of it.
Should we separate float/string PR's? I'm happy with Float implementation and could merge it.

Reply:
Thanks! IMO I too believe the simplest parser should be enough, the user can pre-parse it's input to be valid. A fully compliant parser would be too much for this library scope. I've updated the PR with the suggested changes.
I'll split the PR into float and string tomorrow.

Reply:
We can think on a simple NewFromString(), perhaps only accepting the "." as decimal separator and no other char accepted.

Another function can be `money.Parse(amount string, currency string)` to take into account locale separators.

Reply:
The `NewFromString` function in this PR only accept simple float-like strings, with the only restriction that they have to use the `currency.Decimal` as the separator. But since one must specify a currency to instantiate a Money, I believe this is acceptable. I'm open to change it to only accept "." decimals though, if it makes the function simpler.